tests/lib/snaps/content-slot/meta/snap.yaml

```diff
@@ -0,0 +1,8 @@
+name: content-slot
+version: 1.0
+slots:
+    shared-content-slot:
+        interface: content
+        content: mylib
+        read:
```

Comment:
Given this is always relative to the snap name I wonder if there's any test that we can do for `write`.